.devcontainer.json

```diff
@@ -21,7 +21,11 @@
   ],
   "mounts": [
     // Crosvm wants access to syslog.
```

Comment:
nit: I would suggest removing this comment, or adding another comment as to why the new mounts are needed, otherwise it seems like all of them are related to syslog.

Reply:
Done, added the same comments here as in the docker_run script.